src/components/MultiHopTrade/hooks/useTradeExecution/useTradeExecution.ts

```diff
@@ -102,6 +104,7 @@ export const useTradeExecution = ({
         supportsEIP1559,
         buyAssetUsdRate,
         feeAssetUsdRate,
+        slippageTolerancePercentage: slippagePercentage,
```

Comment:
We may want to bring the decimal vernacular here to make it clear this is the internal decimal representation vs. the user one

Reply:
https://github.com/shapeshift/web/pull/5023/commits/492a59022179229d731392f0e20739e96afdf726